app/controllers/application_controller.rb

```diff
@@ -1,10 +1,9 @@
 require './config/environment'
 
 class ApplicationController < Sinatra::Base
-
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
   end
-
-end
\ No newline at end of file
+end
```

Comment:
Here's where you would define your helpers -- I think `current_user` and `logged_in?` would be particularly helpful.